storage/src/database/mod.rs

```diff
@@ -200,19 +200,19 @@ where
     }
 
     /// Iterator over entries with key starting with given prefix
-    pub fn prefix_iter<Pfx>(&self, prefix: &Pfx) -> crate::Result<impl '_ + EntryIterator<DbMap>>
+    pub fn prefix_iter<Pfx>(&self, prefix: &Pfx) -> crate::Result<impl 'tx + EntryIterator<DbMap>>
```

Comment:
As a matter of style, we usually put the lifetime bound after the trait.

Reply:
I know you didn't change it, but apparently that was an oversight.

Reply:
> As a matter of style, we usually put the lifetime bound after the trait.

FYI: there were  quite a few places like that, I decided to fix them all - https://github.com/mintlayer/mintlayer-core/pull/1769/commits/799373465d992965aa150ff2b2b38cabe43b91e2
(or at least this is what I've found via a regex)